src/components/Errors.tsx

```diff
@@ -0,0 +1,37 @@
+import React, { useEffect } from 'react';
+import classNames from 'classnames';
+import { ErrorMessage } from '../types/ErrorMessage';
+
+type Props = {
+  errorMessage: ErrorMessage,
+  closeError: () => void,
+};
+
+export const Errors: React.FC<Props> = ({ errorMessage, closeError }) => {
+  useEffect(() => {
+    setTimeout(() => {
```

Comment:
Clear the setTimeout by returning a cleanup function in the useEffect to avoid errors when the component unmounts. 🧹

Reply:
not sure that I understand that point correctly